crates/esthri-cli/src/main.rs

```diff
@@ -228,12 +227,17 @@ fn call_real_aws() {
     let aws_tool_path = env::var(REAL_AWS_EXECUTABLE_ENV_NAME)
         .unwrap_or_else(|_| REAL_AWS_EXECUTABLE_DEFAULT.to_string());
 
-    let err = Command::new(&aws_tool_path).args(args).exec();
-
-    panic!(
-        "Executing aws didn't work. Is it installed and available as {:?}? {:?}",
-        aws_tool_path, err
-    );
+    Command::new(&aws_tool_path)
+        .args(args)
+        .stdout(Stdio::inherit())
+        .output()
+        .expect(
+            format!(
+                "Executing aws didn't work. Is it installed and available as {:?} ",
+                aws_tool_path
+            )
+            .as_str(),
+        );
```

Comment:
I think you can do something like 

```suggestion
    let status = Command::new(&aws_tool_path)
        .args(args)
        .stdout(Stdio::inherit())
        .status()
        .expect(
            format!(
                "Executing aws didn't work. Is it installed and available as {:?} ",
                aws_tool_path
            )
            .as_str(),
        );
    std::process::exit(status.code().unwrap_or(-1));
```

to propagate the status code

Reply:
the struct returned by `.output()` also has a `.status()` method which would get you the same thing, but I think status is better here because we are not interested in the output (as in stdout/err)